src/metabase/driver/crate/util.clj

```diff
@@ -12,8 +13,16 @@
     :milliseconds (kutils/func (str "TRY_CAST(%s as TIMESTAMP)") [expr])))
 
 (defn- date-trunc [unit expr]
-  "date_trunc('interval', timestamp): truncates a timestamp to a given interval"
-  (k/sqlfn :DATE_TRUNC (kx/literal unit) expr))
+  "date_trunc('interval', timezone, timestamp): truncates a timestamp to a given interval"
+  (if (or (nil? (driver/report-timezone)) (= (count (driver/report-timezone)) 0))
```

Comment:
@agilliland understand. However, if I use it like that it always returns `nil` although I defined a report timezone in the settings. Accessing the setting directly like I did before worked well.

```clojure
  (let [timezone    (get-in qp/*query* [:settings :report-timezone])]
    (if (= (nil? timezone) true)  ;; this always returns nil
      (k/sqlfn :DATE_FORMAT format expr)
      (k/sqlfn :DATE_FORMAT format timezone expr)))
```